lib/actions.ts

```diff
@@ -23,7 +23,7 @@ export const checkoutCurrentUserWithPlan = async (planId: string) => {
       }
 
   const connectSession: ConnectSessionParams = {
-    callbackUrl: 'http://localhost:3000?error=connectSessionFailed',
```

Comment:
I wonder if it would showcase the usefulness of Connect Sessions more if we redirect to another page in the app. 

Currently, on every example, we just go back to the page the actions was invoked from. This makes sense for all the subscription management specific actions, as there is no other place it makes sense to go to.
However, after buying a subscription, we could redirect to the backoffice page so that I can see the details of the subscription I just bought. This would help demoing how Connect Sessions can help with the flow of user actions in existing applications

Reply:
> I wonder if it would showcase the usefulness of Connect Sessions more if we redirect to another page in the app.

I agree, although we're redirecting to another page already (i.e. `/checkout/callback` instead of just `/checkout`).

> However, after buying a subscription, we could redirect to the backoffice page so that I can see the details of the subscription I just bought. This would help demoing how Connect Sessions can help with the flow of user actions in existing applications

I believe that the callback page does this exact thing; it shows you the phone plan you chose through the `<PurchasedPlanCard />` (do not confuse with `<PurchasePlanCard />` albeit name is almost the same). 

Redirecting to the `/backoffice` doesn't make much sense to me from a user standpoint, as I don't expect the user to have access to the `/backoffice` in the first place :/. 

Is there something you would want to improve around the callback page UI or was it perhaps not obvious that the page was different than the original `/checkout` page (just cause they look similar)? 😄

Reply:
I think you just thinking of two different experiences:

-  usually big online stores and etc. have the "My orders"/ "Purchases"/ "Account" sections where user get the overview of all their stuff, `/backoffice` in our example, so it makes sense to redirect users to such a page
- small businesses powered by Shopify or similar products: they don't have an "Account" section or something like that, users only get the order summary page, which is what`/checkout/callback` looks like

Both are completely valid use cases, although I think we as a company leaning more towards the first one: integrating with existing apps that already have their own user base, I would expect them to have an `/backoffice`-like page. The assumption is:  they want their users to manage subscriptions/addons/etc. in their product, not Connect, hence redirecting straight to `/backoffice` - displaying where in the app a user can find everything they bought and can manage. 

That being said, I don't see why we can't combine a "successful order" summary page with the introduction of "here is all your stuff" concept: if `<PurchasedPlanCard />` had a link to `/backoffice`, it would be clear there's more to explore. It is unfortunate that `/checkout/callback` looks exactly like `/checkout`, though, took me a second to notice the difference😅 I'm also not sure about the email instructions text, I think it's important enough info to be included in the plan card, something like "Plan activation instructions were sent to your email", we shouldn't state they received the email already 🤔 

I also like that this page deals with the error state as well 👍

Reply:
> Is there something you would want to improve around the callback page UI or was it perhaps not obvious that the page was different than the original /checkout page (just cause they look similar)?

Honestly, I completely overlooked that this was a different page when watching the video. The purpose of this demo is to show what Connect Sessions could potentially do for customers and I think this becomes a lot clearer when the flow is `Checkout -> Connect -> Backoffice (completely different page)`. 

This way, it becomes inherently clear that Connect Sessions can start at place A and "end" in place B without having to take a close look at the code or URL

Reply:
Gotcha.

My understanding was that `/backoffice` was more like a "business-side" page (more like what @gigsanna described in the 2nd experience).

If we always felt that `/backoffice` is used-targetted, then I can use that (it's just that `/backoffice` looks a bit weird as a URL; I'd prefer to name it something like `/account` or `/subscription`)

Reply:
renamed to `/phone-plans` to match the sidebar (spoke with @Plsr )

Reply:
Addressed